### PR TITLE
User and Stack Routes Structured

### PR DESCRIPTION
**Pull request by shivamarora96**

### CODE REVIEW

**Comment by shivamarora96** [comment]
#106

**Comment by shivamarora96** [comment]
@tharindupr Please have a look at the PR

**Comment by psnmissaka** [comment]
Fix this warning in the PR as well WARNING: `The useMongoClient option is no longer necessary in mongoose 5.x, please remove it.`

File: stackle_api/app/models/stack.js [hunk 1]
```diff
@@ -4,11 +4,110 @@ const returnWithResponse = require('../lib/returnWithResponse');
...
 
-module.exports = mongoose.model('Stack', stackSchema);
\ No newline at end of file
+
+//create a stack
+stackSchema.statics.createStack = function(request, response){
+	try {
+			const validator = new Validator(request.body);
+			const input = validator.validateCreateStack();
+			console.log('Input : ', input);
```
Comment (psnmissaka) [inline stackle_api/app/models/stack.js]: remove the console log

**Comment by psnmissaka** [comment]
Not related to this issue but the API endpoint `localhost:8080/api/post/create` validator is not working properly. The reason is due to validator checking for an attribute called `link_issue`. The attribute should be `linkIssue`

**Comment by shivamarora96** [comment]
Sure @psnmissaka

**Comment by shivamarora96** [comment]
@psnmissaka  @tharindupr  changes made, kindly review it. #106

**Pull request merged**